internal/backend/sessions/sessionworkflows/workflow.go

```diff
@@ -103,6 +105,8 @@ func (w *sessionWorkflow) addPrint(ctx workflow.Context, print string) {
 func (w *sessionWorkflow) updateState(ctx workflow.Context, state sdktypes.SessionState) {
```

Comment:
I find it confusion that you call the workflow context `ctx` and there's the built-in context which is traditionally called `ctx`. Maybe rename the parameter to `wctx` ?

Reply:
done. i think from now on for workflow context we'll use wctx